src/components/Link/Link.tsx

```diff
@@ -24,7 +24,7 @@ const isFileLink = (str: string) =>
  * @param { string } url
  * @returns { boolean }
  */
-const isExternalLink = (url: string) => {
```

Comment:
Since this is purely a JavaScript utility it would be better off being moved to the utils folder